src/core/vertex.js

```diff
@@ -446,6 +448,7 @@ p5.prototype.curveVertex = function(x, y) {
  *
  */
 p5.prototype.endContour = function() {
+  p5._validateParameters('endContour', arguments);
```

Comment:
are these ones with no parameters necessary?

Reply:
i think so. isn't `endContour('foo')` an error?

Reply:
i don't see anything that would cause this code to break if 'foo' were passed in?

Reply:
that's true. but if someone passed in an argument presumably expecting it to do something, and it didn't, wouldn't it be useful to let them know?

Reply:
please see my comment here https://github.com/processing/p5.js/pull/2807#issuecomment-384143965